src/Sarif.FunctionalTests/SarifConverterTestData/ClangAnalyzer/FileNumberCoverage.xml.sarif

```diff
@@ -5,19 +5,26 @@
       "toolInfo": {
         "name": "Clang"
       },
+      "runInfo": {
+        "fileInfo": {
+          "jcparam.c": [
+            {
+              "uri": "jcparam.c",
+              "mimeType": "text/x-cpp"
+            }
+          ]
+        }
+      },
       "results": [
         {
           "ruleId": "Dead assignment",
           "fullMessage": "Value stored to 'scanptr' is never read",
           "locations": [
             {
-              "analysisTarget": [
-                {
-                  "uri": "jcparam.c",
-                  "mimeType": "text/x-cpp",
-                  "region": {}
-                }
-              ]
+              "analysisTarget": {
+                "uri": "jcparam.c",
+                "region": {}
```

Comment:
I filed sarif-standard/sarif-spec#137, "Clarify handling of missing/empty regions".